typespec-tests/src/main/java/com/cadl/multipart/models/FormData.java

```diff
@@ -151,7 +151,7 @@ public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
         jsonWriter.writeIntField("resolution", this.resolution);
         jsonWriter.writeStringField("type", this.type == null ? null : this.type.toString());
         jsonWriter.writeJsonField("size", this.size);
-        jsonWriter.writeUntypedField("image", this.image == null ? null : this.image.toObject(Object.class));
```

Comment:
will we have issue if we remove null protection?

Reply:
The `image` property is required in tsp, so it should not be `null`. User will get a null exception if they pass a null, but I guess that is better than silent ignore.